### PR TITLE
Clarify minimum transaction error message

### PR DESCRIPTION
**Pull request by z64**

### CODE REVIEW

File: spec/stackcoin/bank_spec.cr [hunk 1]
```diff
@@ -9,7 +9,7 @@ daniel_id = 3_u64
 def create_empty_test_bank
   db = DB.open "sqlite3://%3Amemory%3A"
   StackCoin::Database.new StackCoin::Config.new, db
-  StackCoin::Bank.new db
+  StackCoin::Bank.new db, StackCoin::Banned.new(db)
```
Comment (jackharrhy) [inline spec/stackcoin/bank_spec.cr:12]: man i gotta resolve #35

File: src/stackcoin/bank.cr [hunk 2]
```diff
@@ -119,7 +119,7 @@ class StackCoin::Bank
 
   def transfer(from_id : UInt64, to_id : UInt64, amount : Int32)
     return Result::TransferSelf.new "Can't transfer money to self" if from_id == to_id
-    return Result::InvalidAmount.new "Amount can't be less than zero" if amount <= 0
+    return Result::InvalidAmount.new "Amount must be greater than zero" unless amount > 0
```
Comment (jackharrhy) [inline src/stackcoin/bank.cr:122]: i wrote most of this code before i knew of unless, i should move more code over to use its _clean_ and more english like

**Comment by jackharrhy** [comment]
LGTM
image

File: spec/stackcoin/bank_spec.cr (hunk 1 above)
Comment (z64) [reply spec/stackcoin/bank_spec.cr:12]: image

**Pull request merged**